Source/Dafny/DafnyPipeline.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <Target Name="PreBuild" BeforeTargets="PreBuildEvent">
-    <Exec Command="dotnet tool restore &amp;&amp; dotnet tool run coco $(ProjectDir)Dafny.atg -namespace Microsoft.Dafny -frames $(ProjectDir)../../third_party/Coco/src" />
+    <Exec Command="dotnet tool restore &amp;&amp; sleep 1 &amp;&amp; dotnet tool run coco $(ProjectDir)Dafny.atg -namespace Microsoft.Dafny -frames $(ProjectDir)../../third_party/Coco/src" />
```

Comment:
Does it still make sense to split these into 2 commands (or 3, with the `sleep`)? That way, the logs will more clearly indicate which of the two is failing.